src/editor/exampleDoc.js

```diff
@@ -1,6 +1,7 @@
 import pushID from "../pushID";
```

Comment:
Out of curiosity, how did you notice this? (Asking because it's easy to miss).

Were you actively checking for // @flow at the top of each file, while reviewing (because you know this PR is for flow type)s?

Reply:
I knew to look for it — I've been caught out by it many times before. 😀

There's an ESLint plugin that includes a check for the file annotation, but let's hold off on that until we're using Flow a bit more widely:

https://github.com/gajus/eslint-plugin-flowtype#require-valid-file-annotation